src/main/java/com/treasuredata/client/model/TDTableDistribution.java

```diff
@@ -0,0 +1,72 @@
+package com.treasuredata.client.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.base.Objects;
+
+/**
+ *
+ */
+public class TDTableDistribution
+{
+    private final long userTableId;
+    private final long bucketCount;
+    private final String partitionFunction;
```

Comment:
@Lewuathe We also need to know which column(s) are used for partitioning.  Add column names and their types so that we can apply an appropriate bucketing function in td-spark.

Reply:
Ah, good catch. True. Will do.